CHANGELOG.md

```diff
@@ -45,6 +45,8 @@ can cause a deadlock to occur over `logging._lock` in some cases ([#4636](https:
   ([#4669](https://github.com/open-telemetry/opentelemetry-python/pull/4669))
 - Set expected User-Agent in HTTP headers for grpc OTLP exporter
   ([#4658](https://github.com/open-telemetry/opentelemetry-python/pull/4658))
+- Remove LogData and extend SDK LogRecord to have instrumentation scope
```

Comment:
We should have a breaking changes section or something here, maybe I can update the title of the PR to reflect that